articles/connections/social/fitbit.md

```diff
@@ -4,15 +4,17 @@ image: /media/connections/fitbit.png
 seo_alias: fitbit
 ---
 
-# Obtain a *Consumer Key* and *Consumer Secret* for Fitbit
+# Obtain a *Client ID* and *Client Secret* for Fitbit
 
-To configure a Fitbit oAuth connection, you will need to register a new application in Fitbit.
+**Fitbit has ended support for OAuth 1.0. New connections with Fitbit will use OAuth 2.0. Please see the following for reference on OAuth 2.0 at Fitbit: [https://dev.fitbit.com/docs/oauth2/](https://dev.fitbit.com/docs/oauth2/).**
```

Comment:
I think a warning panel works best for this

::: panel-warning Fitbit has ended support for OAuth 1.0
New connections with Fitbit will use OAuth 2.0. Please see the following for reference on OAuth 2.0 at Fitbit: [https://dev.fitbit.com/docs/oauth2/](https://dev.fitbit.com/docs/oauth2/)
:::